psse/psse-model/src/main/java/com/powsybl/psse/model/pf/PssePowerFlowModel.java

```diff
@@ -105,6 +113,22 @@ public void addAreas(List<PsseArea> areas) {
         return Collections.unmodifiableList(areas);
     }
 
+    public void addTransformerImpedanceCorrections(List<PsseTransformerImpedanceCorrection> transformerImpedanceCorrections) {
```

Comment:
Should you check that the list is not `null`? Same thing for other methods in the file

Reply:
We believe that is not necessary as these methods are a simple call to the `adAll` method and the `adAll` method generates a `NullPointerException`  if the specified list is `null`